config/jobs/kubernetes-sigs/kind/kind-presubmits.yaml

```diff
@@ -242,6 +242,56 @@ presubmits:
             # during the tests more like 3-20m is used
             cpu: 2000m
   # mimic pull-kubernetes-e2e-kind, but using kind built in this PR
+  # conformance test against kubernetes release-1.19 branch with `kind`, skipping
+  # serial tests so it runs in ~20m
+  - name: pull-kind-e2e-kubernetes-1-19
+    always_run: true
+    labels:
+      preset-bazel-scratch-dir: "true"
+      preset-bazel-remote-cache-enabled: "true"
+      preset-dind-enabled: "true"
+      preset-kind-volume-mounts: "true"
+    decorate: true
+    extra_refs:
+    - org: kubernetes
+      repo: kubernetes
+      base_ref: release-1.19
+      path_alias: k8s.io/kubernetes
+    path_alias: sigs.k8s.io/kind
+    decoration_config:
+      timeout: 40m
+      grace_period: 15m
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/krte:v20200824-5d057db-1.19
+        command:
+        - wrapper.sh
+        - bash
+        - -c
+        - cd ./../../k8s.io/kubernetes && ./../../sigs.k8s.io/kind/hack/ci/e2e.sh
+        env:
+        - name: FOCUS
+          value: "."
+        # TODO(bentheelder): reduce the skip list further
+        # NOTE: changes should filter down to pull-kubernetes-e2e-kind-canary
+        # and then pull-kubernetes-e2e-kind
+        - name: SKIP
+          value: \[Slow\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]|PodSecurityPolicy|LoadBalancer|load.balancer|In-tree.Volumes.\[Driver:.nfs\]|PersistentVolumes.NFS|Network.should.set.TCP.CLOSE_WAIT.timeout|Simple.pod.should.support.exec.through.an.HTTP.proxy|subPath.should.support.existing|should.provide.basic.identity|\[NodeFeature:PodReadinessGate\]
```

Comment:
I don't think this matches upstream 1.19?

Reply:
it's the same we have for 1.18 :thinking:

Reply:
https://github.com/kubernetes/test-infra/blob/1f6f59015c99771a6e359850b5b3ef1b7bb979fb/config/jobs/kubernetes/sig-release/release-branch-jobs/1.19.yaml#L1432

Reply:
someday we will move this out of prow config ..